modules/ServerAPI/src/server/routes/FileServices.coffee

```diff
@@ -67,14 +123,13 @@ setupRoutes = (app, loginRoutes, requireLogin) ->
 					console.error(err)
 					resp.send(err)
 
-
 	app.post '/blobUploads', getBlobByteArray
 
-	upload.on "error", (e) ->
-		console.log "fileUpload: ", e.message
-	upload.on "end", (fileInfo) ->
-		console.log fileInfo
-		app.emit "file-uploaded", fileInfo
+	# upload.on "error", (e) ->
```

Comment:
Can you just delete these comments?

Reply:
I just removed those dead lines. Will remove from package.json.